docs/docker.md

```diff
@@ -107,9 +107,8 @@ introduction to Docker.
 -   The `MAINTAINER` instruction is deprecated. The `LABEL` instruction is a much 
     more flexible version of this and you should use it instead, as it enables 
     setting any metadata you require, and can be viewed easily, for example 
-    with docker inspect. Dockerfiles **SHOULD** put the labels `maintainer`, 
-    `release-date`, `version`, and `vendor`.
-
+    with docker inspect. See section [Label description content](#label-description-content) 
```

Comment:
> docker inspect

Add back ticks to monospace  => `docker inspect`

Reply:
Done